manifests/s/SoftMaker/FreePDF/2022/3.0.8/SoftMaker.FreePDF.2022.locale.en-US.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: SoftMaker.FreePDF.2022
+PackageVersion: 3.0.8
+PackageLocale: en-US
+Publisher: SoftMaker Software GmbH
+PackageName: FreePDF 2022
+License: Proprietary
+Copyright: SoftMaker Software GmbH

```

```suggestion
Copyright: Copyright (c) 2022 SoftMaker
```
Got it from the package's website